src/glmtools.jl

```diff
@@ -103,6 +103,25 @@ A [`Link`](@ref) defined as `η = √μ`
 """
 struct SqrtLink <: Link end
 
+"""
+    PowerLink
+
+A [`Link`](@ref) defined as `η = log(μ)` when `λ = 0`, otherwise `η = μ^λ`.
+
+PowerLink refers to a class of techniques that use a power function
+(like a logarithm or exponent) to transform the responses to
+Gaussian or more-Gaussian like.
```

Comment:
I don't understand why you mention exponent here. Is that supported?

Reply:
Rephrased as `PowerLink refers to the class of transforms that use a power function or logarithmic function to transform responses into a Gaussian or a Gaussian-like.`